pkg/standalone/standalone.go

```diff
@@ -225,8 +226,7 @@ func makePodSourceConfig(kc *KubeletConfig) *config.PodConfig {
 		glog.Infof("Adding manifest url: %v", kc.ManifestURL)
 		config.NewSourceURL(kc.ManifestURL, kc.HttpCheckFrequency, cfg.Channel(kubelet.HTTPSource))
 	}
-
-	if kc.EtcdClient != nil {
+	if !reflect.ValueOf(kc.EtcdClient).IsNil() {
```

Comment:
Sure that makes sense.

Though, if you ask me, it doesn't make sense to have an interface `(*<anytype>, nil) != nil` since the bug of returning a type and assigning it to an interface making nil become non-nil is also subtle.

I'll send a PR to undo this use of `reflect` and return the interface from `kubelet.EtcdClientOrDie()`.

Cheers,
Filipe